mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/impl/MessageParserTest.java

```diff
@@ -51,6 +51,14 @@ public void getAttachmentsShouldRetrieveAttachmentsWhenOne() throws Exception {
     }
 
     @Test
+    public void getAttachmentsShouldReturnTheExpectedAttachment() throws Exception {
+        List<Attachment> attachments = testee.retrieveAttachments(ClassLoader.getSystemResourceAsStream("eml/oneAttachmentAndSomeInlined.eml"));
+
+        Attachment attachment = attachments.get(0);
+        assertThat(attachment.getStream()).hasContentEqualTo(ClassLoader.getSystemResourceAsStream("eml/exploits_of_a_mom.png"));
+    }
+
+    @Test
     public void getAttachmentsShouldRetrieveAttachmentsWhenTwo() throws Exception {
         List<Attachment> attachments = testee.retrieveAttachments(ClassLoader.getSystemResourceAsStream("eml/twoAttachments.eml"));
 
```

Comment:
This was embedded in one of the emails. Do you think we should find other emails examples?